spec/features/cases/offender_sar_complaint/adding_note_to_case_history_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+feature 'When viewing an offender sar case' do
```

Comment:
Small thing but may be confusing in future - we should probably change "offender sar case" to "offender sar complaint case" here. Possibly in the from "case" to "complaint" in the file name as well?